src/container.ts

```diff
@@ -215,6 +215,17 @@ container.register({
   [SPID_STRATEGY]: awilix.asFunction(spidStrategy).singleton()
 });
 
+// Set default idp metadata refresh time to 10 days
+export const DEFAULT_IDP_METADATA_UPDATE_TIME = 3600 * 24 * 10;
```

Comment:
always specify the unit in the identifier name:

```suggestion
export const DEFAULT_IDP_METADATA_UPDATE_INTERVAL_SECONDS = 3600 * 24 * 10;
```

Reply:
fixed in 82e8752